components/bl-copyButton-component/src/index.js

```diff
@@ -0,0 +1,36 @@
+import { useState, useMemo } from 'react';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function CopyButton({ component, eventHandlers }) {
+  const { display, classList, isDisabled, content, copyLabel, copiedLabel, copiedDuration } = component;
+  const { onCopy } = eventHandlers;
+
+  const [isCopied, setIsCopied] = useState(false);
+
+  component.copy = () => {
+    navigator.clipboard.writeText(content).then(() => {
+      setIsCopied(true);
+
+      setTimeout(() => {
+        setIsCopied(false);
+      }, copiedDuration);
+    });
+  };
+
+  if (!display) {
+    return null;
+  }
+
+  return (
+    <div className={ cn('bl-customComponent-copyButton', classList) }>
```

Comment:
We need this wrapper? That may cause some problems when Vladimir added default stylable properties